app/crud.py

```
@@ -0,0 +1,65 @@
+    except Exception as e:
+        await db.rollback()
+        raise HTTPException(status_code=500, detail=f"Error creating ToDo: {e}") from e
+
+
+async def get_todos(db: AsyncSession) -> list[ToDo]:
+    try:
+        result = await db.execute(select(ToDo))
+        return result.scalars().all()
+    except Exception as e:
+        raise HTTPException(
+            status_code=500, detail=f"Error retrieving ToDos: {e}"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)